components/camel-aws/src/main/java/org/apache/camel/component/aws/sqs/SqsComponent.java

```diff
@@ -62,6 +62,10 @@ protected Endpoint createEndpoint(String uri, String remaining, Map<String, Obje
             throw new IllegalArgumentException("Extending message visibility (extendMessageVisibility) requires visibilityTimeout to be set on the Endpoint.");
         }
         
+        if (configuration.isFifoQueue() && configuration.getMessageGroupIdStrategy() == null) {
```

Comment:
Use ObjectHelper here instead of null checking

Reply:
Will change to `ObjectHelper.isEmpty(configuration.getMessageGroupIdStrategy())`